doc/man7/flux-environment.rst

```diff
@@ -426,6 +426,15 @@ MISCELLANEOUS
    command front end executable used by the ssh connector to start
    :program:`flux relay` on the remote system.
 
+.. envvar:: DBUS_SESSION_BUS_ADDRESS
+
+   :man1:`flux-exec` sets this to point to the Flux instance owner's
+   D-Bus instance, to ensure that a remote invocation of
+   :option:`systemctl --user` accesses Flux's service manager.  This is
```

Comment:
Should this be "accesses the service manager for the instance user"?

Reply:
Fixed - er, I used "for the Flux instance owner".  Hope that's OK.